crates/core/src/blob.rs

```diff
@@ -66,17 +66,46 @@ impl<T: Default> Initialize<T> for BlobTypeMap<T> {
     }
 }
 
+new_id!(BlobId, "blob");
+
+pub trait PackedId: Copy + Into<BlobId> + From<BlobId> {
```

Comment:
Also here, does it need to be part of the public API?

Reply:
Yes, it is used in `Repository::get_index_item`.

Reply:
Do we then need to export it directly in `lib.rs`?